tests/Data/ProgrammesDb/EntityRepository/ServiceRepository/FindByIdsWithNetworkServicesListTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ServiceRepository;
+
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ServiceRepository;
+use BBC\ProgrammesPagesService\Domain\ValueObject\Sid;
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ServiceRepository::<public>
+ */
+class FindByIdsWithNetworkServicesListTest extends AbstractDatabaseTest
+{
+    /** @var ServiceRepository */
+    private $repo;
+
+    public function setUp()
+    {
+        $this->loadFixtures(['NetworksFixture']);
+        $this->repo = $this->getEntityManager()->getRepository('ProgrammesPagesService:Service');
+    }
+
+    public function testFindServiceBy()
+    {
+        $sid = new Sid('bbc_radio_fourlw');
+        $dbId = $this->getDbId($sid);
```

Comment:
Use the utility functions rather than wrangling getSQLLogger bits yourself:

Also you don't need the Sid value object at all as this is a DB query so it's all strings.

```
$dbId = $this->getDbIdFromPersistentIdentifier('bbc_radio_fourlw', 'Service', 'sid');
```

Or actually your Service fixture has a PID so use that instead of the Sid (as that's the id that `getDbIdFromPersistentIdentifier` defaults to using):

```
$dbId = $this->getDbIdFromPersistentIdentifier('p00fzl7k', 'Service');
```